exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -97,19 +97,6 @@
     };
 
 </script>
-<style>
```

Comment:
Sorting style is removed and `<link href="/static/css/drill-dataTables.sortable.css" rel="stylesheet">` is not added. Is it missed or it was done for intention?

Reply:
Since `sortable` class is used in script in `profile.ftl` I think link to css file is missed.

Reply:
Yes, I missed it and forgot to test this page.